website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1097,7 +1097,7 @@ The `/sys/internal/counters/config` endpoint is used to configure logging of act
 
 ### Parameters
 
-- `default_report_months` `(integer: 12)` - The number of months to report if no `start_time` is specified in a query.
+- `default_report_months` `(integer: 12)` - (*Note: This field is deprecated*) The number of months to report if no `start_time` is specified in a query.
```

Comment:
@schavis do you have any specific style guidance for deprecated fields?

Reply:
Added a suggestion :)